### PR TITLE
lambdatest-upload 1.0.1

### PR DESCRIPTION
**Pull request by swngarg-lt**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by rohithmone** [comment]
@tothszabi We have made few changes to the older version. Please review this PR. Thankyou

File: steps/lambdatest-upload/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      The access key you use to log into your LambdaTest account
+    is_required: true
+    is_sensitive: true
+    summary: Your LambdaTest access key
+    title: LambdaTest access key
+- app_name: null
+  opts:
+    description: |
+      The name of your application
+    is_required: true
```
Comment (mateherber) [inline steps/lambdatest-upload/1.0.1/step.yml:67]: Adding a new `required` input with no default value is considered a "breaking change" so according to semver the version of the step will have to be `2.0.0`

**Review by mateherber — CHANGES_REQUESTED** [review]

**Comment by mateherber** [comment]
Hey @rohithmone! 

Thanks for adding changes to your step, please read my review remark.

**Comment by rohithmone** [comment]
Made the respective changes @mateherber

**Review by mateherber — APPROVED** [review]

**Pull request merged**